### PR TITLE
Get rid of redundant projection matrix computations in Splatfacto (needs a gsplat update)

### PR DESCRIPTION
**Pull request by oseiskar**
Requires https://github.com/nerfstudio-project/gsplat/pull/97

### CODE REVIEW

File: nerfstudio/models/splatfacto.py
Comment (kerrj) [inline nerfstudio/models/splatfacto.py:717]: If https://github.com/nerfstudio-project/gsplat/pull/149 gets merged then this needs to be updated, I think just changing the API all at once one time is better than 2 incremental changes

**Comment by oseiskar** [comment]
> One more thing, the gsplat version should be bumped to `gsplat>=0.1.9` in pyproject.toml

Done!

**Comment by jb-ye** [comment]
We need to bump up gsplat version first before we can merge this PR.

**Review by kerrj — APPROVED** [review]
lgtm! will merge once gsplat is bumped

**Pull request merged**
